ef_tx.go

```diff
@@ -0,0 +1,68 @@
+package bux
+
+import (
+	"context"
+	"encoding/hex"
+
+	"github.com/libsv/go-bt/v2"
+)
+
+func ToEfHex(ctx context.Context, tx *Transaction, store TransactionGetter) (efHex string, ok bool) {
```

Comment:
Minor - lint error:
`exported: exported function ToEfHex should have comment or be unexported (revive)go-golangci-lint`

Reply:
Done